### ISSUE TITLE
Multi Fallback Language

### ISSUE THREAD

**Issue by EslaM-M** [isaachinman/next-i18next#1210]
## Describe the bug
The issue is when define fallbackLng as an object with multiple fallback options deson't work 
`  fallbackLng: {
      'en-eg': ['en'],
  }`
  it only works if I defined the fallbackLng as a string like 
  fallbackLng: 'en'

after some investigation on the library itself i found this block of code 
`  if (typeof config.fallbackLng === 'string') {
              initialI18nStore[config.fallbackLng] = {};
            }`
which means the fallback language assigned if it's a string only.
## Occurs in next-i18next version
8.2.0

## Steps to reproduce

- in the next-i18next.config.js file add this as i18n  object 
```
module.exports = {
  i18n: {
    locales: ['en-eg', 'ar-eg', 'en', 'ar'],
    defaultLocale: 'en-eg',
  },
  supportedLngs: ['en-eg', 'ar-eg', 'en', 'ar'],
  debug: true,
  lowerCaseLng: true,
  defaultLanguage: 'en-eg',
  otherLanguages: ['ar', 'en', 'en-eg', 'ar-eg'],
  whitelist: ['ar', 'en', 'en-eg', 'ar-eg'],
  fallbackLng: {
    'en-eg': ['en'],
    'ar-eg': ['ar'],
  },
};
```

- add the local files in the following path

![test](https://user-images.githubusercontent.com/7307221/117303711-4c595500-ae7d-11eb-92d5-0c8dc2bcc89f.png)

try to use a translation key that exists on the en/common.json while your language is
en-eg. you will find that the key is not translated hover it's defined as a fallback language
`

## Expected behavior
if  you define a fallback language for a specific language it should be working probably 

## Screenshots
If applicable, add screenshots or a GIF to help explain your problem.

## OS (please complete the following information)
 - Browser: [e.g. Chrome  89.0.4389.90 ]

**Comment by isaachinman** [comment]
I believe this is fixed by #1197, right?

**Comment by EslaM-M** [comment]
@isaachinman yes it's fixed :+1:

**Comment by Nikoms** [comment]
@isaachinman do you plan to create a new release with this fix soon? 

Thanks :)

**Comment by isaachinman** [comment]
Released in v8.4.0.
